compat/js/siteorigin-panels-layout-block.jsx

```diff
@@ -118,6 +118,12 @@ class SiteOriginPanelsLayoutBlock extends Component {
 		} );
 		
 		jQuery( document ).trigger( 'panels_setup', this.builderView );
+
+		if ( typeof window.soPanelsBuilderView == 'undefined' ) {
+			window.soPanelsBuilderView = [];
+		}
```

Comment:
It looks like something strange is going on with this line. Is this closing `}` meant to be here?

Reply:
Yes. If `window.soPanelsBuilderView` isn't initialized, this creates it as an empty array.